cmd/flags.go

```diff
@@ -206,3 +229,14 @@ func EnvOrDefault(key string, defaultVal string) string {
 	}
 	return defaultVal
 }
+
+func defaultPath(placeholderPath, platformAPI, layersDir string) string {
+	fileName := filepath.Base(placeholderPath)
```

Comment:
It might be simpler just to make more constants along the lines of `DefaultGroupFile` rather than splitting it out from the Placeholder. You could even use those constants to create the placeholders
```
PlaceholderGroupPath           = filepath.Join("<layers>", DefaultGroupFile)
```

Reply:
I don't have strong opinion about any of these approaches so I'll make this change.
Thanks!